_projects/2023/100481838/100481838.Rmd

```diff
@@ -0,0 +1,1067 @@
+---
+title: "We feel it already. How about we visualize it?"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Mikaela De Smedt
+date: 2023-12-16
```

Comment:
Please use ```date: "`r Sys.Date()`"``` instead. The date will be automatically generated when the document is knitted.